### PR TITLE
Added tool sample for PageController

### PR DESCRIPTION
**Pull request by Nash0x7E2**
## Description

* Added code sample for `PageController` similar to `TabController`.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/page_view.dart [hunk 1]
```diff
@@ -36,6 +36,81 @@ import 'viewport.dart';
...
+///
+///   @override
+///   void dispose() {
+///     _pageController.dispose();
+///     super.dispose();
+///   }
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return PageView(
```
Comment (Piinks) [inline packages/flutter/lib/src/widgets/page_view.dart]:
```suggestion
///     return Directionality(
           textDirection: TextDirection.ltr,
           child: PageView(
```
When I ran the sample code, there was an error produced from not having a `Directionality` error available. This was easily fixed just by wrapping everything up in one.

File: packages/flutter/lib/src/widgets/page_view.dart [hunk 2]
```diff
@@ -36,6 +36,81 @@ import 'viewport.dart';
...
+///                     curve: Curves.easeInOut,
+///                   );
+///                 }
+///               },
+///               child: Text('Previous'),
+///             ),
+///           ),
+///         ),
+///       ],
+///     );
```
Comment (Piinks) [inline packages/flutter/lib/src/widgets/page_view.dart]:
```suggestion
///      ),
///     );
```
Additional closing parentheses for the addition of the `Directionality` widget.

**Review by Piinks — COMMENTED** [review]
Hi @Nash0x7E2!
Thank you for the contribution! 🎉
This is a great example! I ran the code and found that this is missing a `Directionality` widget when run by itself. I made a note below on where you could add one to resolve the error. 
Let me know if I can help you resolve it so we can land this contribution! Cheers!

**Comment by Nash0x7E2** [comment]
Hi @Piinks 
I fixed the text directionality bug by adding a `MaterialApp` and a `Scaffold` to the sample.

**Review by Piinks — APPROVED** [review]
LGTM! Thanks again for adding this! 
I'll merge it into master once the tests have completed!

**Comment by Nash0x7E2** [comment]
Awesome! I'm looking forward to adding some more sample very soon.

**Comment by Piinks** [comment]
So happy to hear that! Here is [some more information](https://github.com/flutter/flutter/blob/master/dev/snippets/README.md) on the sample code blocks that we just recently updated if you are interested. I look forward to seeing more!  💯

**Pull request merged**